content/docs/command-reference/push.md

```diff
@@ -94,8 +94,7 @@ in the cache (compared to the default remote.) It can be used to see what files
 - `-j <number>`, `--jobs <number>` - parallelism level for DVC to upload data to
   remote storage. The default value is `4 * cpu_count()`. For SSH remotes, the
   default is `4`. Note that the default value can be set using the `jobs` config
-  option with `dvc remote modify`. Using more jobs may improve the overall
-  transfer speed.
+  option with `dvc remote modify`. Using more jobs may speed up the operation.
```

Comment:
same here, guys, let's try our very very best to keep PRs very focused

Reply:
This was me. Ok but I feel like the alternative is to merge this just for get* and then another PR for the other 6 files. If you prefer that for reviewing, then sure. I honestly thought that alternative was about the same as this single one.

Reply:
my take - it's (almost) always better to do a separate PR. In this case, yes either keep the older version of the language even for the new stuff and merge faster, or fix it and do a separate PR if it's too off).

otherwise it explodes in size, we make it harder to review, title becomes wrong and confuses people, we take longer to merge, etc, etc

Reply:
OK I extracted the sync command changes from this PR and into #2145. Left import* cmds since there's some other reviews going on about them here — or should I move them over too?

Reply:
I guess it's fine in this case.